dotnet-vnc/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full-vnc:latest
+
+RUN wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg && \
+    sudo mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/ && \
+    wget -q https://packages.microsoft.com/config/debian/9/prod.list && \
+    sudo mv prod.list /etc/apt/sources.list.d/microsoft-prod.list && \
+    sudo chown root:root /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
+    sudo chown root:root /etc/apt/sources.list.d/microsoft-prod.list && \
```

Comment:
Nit: You could add `USER root` above these lines, and remove all occurrences of `sudo` and the two `chown` instructions.